code/modules/shuttles/marine_ferry.dm

```diff
@@ -540,16 +540,15 @@
 				break
 		sleep(1)
 
-	for(var/i in GLOB.alive_human_list) //knock down mobs
-		var/mob/living/carbon/human/M = i
-		if(M.z != T_trg.z) continue
-		if(M.buckled)
-			to_chat(M, SPAN_WARNING("You are jolted against [M.buckled]!"))
-			shake_camera(M, 3, 1)
+	for(var/mob/living/carbon/affected_mob in (GLOB.alive_human_list + GLOB.living_xeno_list)) //knock down mobs
+		if(affected_mob.z != T_trg.z) continue
```

Comment:
```suggestion
		if(affected_mob.z != T_trg.z) 
			continue
```
no in line control statements

Reply:
Woops, changed the variable without moving it down, I blame the old coder *dab